manim/__main__.py

```diff
@@ -111,13 +111,16 @@ def get_scene_classes_from_module(module):
 
 def get_module(file_name):
     if file_name == "-":
+        # Since this feature is used for rapid testing, using Scene Caching would be a
+        # hindrance in this case.
+        file_writer_config["disable_caching"] = True
```

Comment:
Comment: after this config system is refactored, we should come back here and use `tempconfig`

Reply:
What do you mean ? I don't see how tempconfig could be used here. It's not, well, a temporary config (in my mind)